### PR TITLE
added: new cards

### PR DESCRIPTION
**Pull request by KaustubhM1997**
Worked on cards from scratch
Added introduction

Live preview - https://astounding-gaufre-a2473e.netlify.app/

### CODE REVIEW

File: CSS/aside.css [hunk 1]
```diff
@@ -10,6 +11,9 @@
   flex: 0 1 16rem;
   border-right: 1px solid var(--secondary-color);
   padding-bottom: 1rem;
+  /* position: fixed; */
+  /* overflow-y: auto;
```
Comment (mimansha-swarup) [inline CSS/aside.css]: remove unnecessary comments

File: CSS/landingpage.css [hunk 2]
```diff
@@ -72,8 +72,8 @@ body {
 }
 
 .main-text {
-  font-size: 1.5rem;
-  max-width: 500px;
+  font-size: 1.8rem;
+  max-width: 700px;
```
Comment (mimansha-swarup) [inline CSS/landingpage.css]: i guess for more than 10 px we should use rem

**Review by mimansha-swarup — APPROVED** [review]

**Review by preethasri — APPROVED** [review]
good to merge.UI looks awesome

**Comment by KaustubhM1997** [comment]
test video 
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 99.0.4844.84 on Windows 10
Viewport:  1536 x 714 @1.25x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-19--astounding-gaufre-a2473e.netlify.app%2F&os=Windows&os_version=10&browser=Chrome&browser_version=99.0&resolution=1536x714&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-19--astounding-gaufre-a2473e.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6248241fcbcf98000896cd1a&commentID=62482ed186b9d24d9f3de9f0&resolution=resolved)

**Pull request merged**